services/src/main/java/com/linkedin/kmf/services/configs/ConsumeServiceConfig.java

```diff
@@ -42,6 +42,10 @@
   public static final String CONSUMER_PROPS_CONFIG = "consume.consumer.props";
   public static final String CONSUMER_PROPS_DOC = "The properties used to config consumer in consume service.";
 
+  public static final String LATENCY_SLA_MS_CONFIG = "consume.latency.sla.ms";
```

Comment:
I see. Thanks for the explanation. I suppose that user will pass in a flat map property file so we need the service name to distinguish the configurations with the same name in different services. Otherwise the patch LGTM.